src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -976,22 +962,23 @@ def customizePerJob(self, job):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
         # HighIO jobs
-        highIOjob = job.get('highIOjob')
-        if highIOjob:
+        if job.get('highIOjob'):
             jdl.append('+Requestioslots = 1\n')
+        else:
+            jdl.append('+Requestioslots = 0\n')
 
         # Performance estimates
-        if job.get('estimatedJobTime'):
-            jdl.append('+MaxWallTimeMins = %d\n' % int(job['estimatedJobTime'] / 60.0))
         if job.get('estimatedMemoryUsage'):
             jdl.append('request_memory = %d\n' % int(job['estimatedMemoryUsage']))
         if job.get('estimatedDiskUsage'):
             jdl.append('request_disk = %d\n' % int(job['estimatedDiskUsage']))
 
-        # Set up JDL for multithreaded jobs
-        if job.get('numberOfCores', 1) > 1:
-            jdl.append('machine_count = 1\n')
-            jdl.append('request_cpus = %s\n' % job.get('numberOfCores', 1))
+        # Max wall time estimate (12h if undefined)
+        jdl.append('+MaxWallTimeMins = %d\n' % int(job.get('estimatedJobTime', 12*3600))/60)
```

Comment:
And here you changed the logic according to the original code `int(job['estimatedJobTime'] / 60.0)`